main.py

```diff
@@ -63,6 +63,8 @@ def log(message, level="info"):
     "region=the_brothers_of_auralia": "TEO",
     "lilystates.proboards.com": "Lily",
     "region=lone_wolves_united": "LWU",
+    "region=warzone_trinidad": "WZT",
+    "region=lair_of_the_dragon": "LotD",
```

Comment:
I would also flag the region tags here, e.g. [region]lair of the dragon[/region].

Reply:
Have flagged LotD. Due to the nature of the Mafia, I didn’t flag [region]WZT[/region], as I never use it when making my WFE’s